mgmtsystem_action/mgmtsystem_action.py

```diff
@@ -45,6 +45,18 @@ class mgmtsystem_action(models.Model):
     system_id = fields.Many2one('mgmtsystem.system', 'System')
     company_id = fields.Many2one('res.company', 'System', default=own_company)
 
+    stage_id = fields.Many2one(
+        'mgmtsystem.action.stage',
+        'Stage',
+        default=lambda self: self.get_default_stage()
+    )
+
+    @api.model
+    def get_default_stage(self):
+        return self.env['mgmtsystem.action.stage'].search([
+            ('is_starting', '=', True)
+        ]).id
```

Comment:
@max3903  Most common notifications are already triggered by mail_message tracking, and it's trivial to add additional messages or actions using Automated Actions. I have been using that extensively to: send reminder notifications, automatically route/assign issues to people, change kanban card colors to yellow or red or their priority, change block user actions on certain conditions.

I can share these implementation if you like. So I still don't see a use case where Workflows would be needed.